### ISSUE TITLE
[BUG] Cellpose training returns no masks after successful training

### ISSUE THREAD

**Issue by eschbachj** [MouseLand/cellpose#1033]
I have trained a new model in the GUI, starting with the cyto model, and it has worked relatively well. I wanted to do further training, so I trained it again, and that worked as well. However, when I went to train it a 3rd or 4th time, it completed the training but says it found no masks. I am testing on the training data, so it isn't an issue of over training. Additionally, I attempted to then train a new model, also starting with the cyto model, on the same training dataset; however, this new model also finds no masks. What could be causing this? I am on an M2 Macbook, macOS Sequoia 15.0, running Cellpose 3.1.0 with Python 3.8. I am working with bright-field microscopy images.

I have seen the previous posts where it was suggested to increase the number of epochs to 1000+, which I tried but it also did not work. I also tried upgrading pytorch to 2.4 as suggested somewhere else, and that also did not work. I tried deleting all the hidden cellpose files and folders, removing my conda environment, recreating the environment, reinstalling cellpose, but still nothing is working. Additionally, the training is extremely slow now even though I have not changed my training dataset. Any help or suggestions would be greatly appreciated!

**Comment by carsen-stringer** [comment]
I'm wondering if the new Mac MPS training isn't working for you? Could you try turning it off with the use GPU checkbox in the GUI (this will enable the CPU)? Also if it's easy could you see for example if cellpose2 releases are actually working better for training? there could be a bug on our end

**Comment by eschbachj** [comment]
I have the GPU checked off always (it never worked for me), and I had the same issue when I tried running the trained model on cellpose2 (and the same issue when trying to train a new model as well).

**Comment by carsen-stringer** [comment]
shoot okay, so the issue only appears with many images?

**Comment by eschbachj** [comment]
I have only used between 2-4 training images, each with between 20-150 masks.

**Comment by carsen-stringer** [comment]
thanks for the info, so to clarify, if you train with e.g. 2 images in your folder it works fine but if you train with all 4 images it fails?

**Comment by eschbachj** [comment]
Hey Carsen, yes, it fails when I click train again multiple times on the exact same image set.

**Comment by eschbachj** [comment]
I tried just training on a very small cropped version of my images, so that the cells take up more of the image. This seems to have fixed the issue!
